server/database/queries/cohort/index.js

```diff
@@ -1,8 +1,11 @@
+const { getCohorts } = require('./getCohorts');
```

Comment:
If you edited `getCohorts` export you should edit this require either.
```suggestion
const getCohorts  = require('./getCohorts');
```

Reply:
:+1:  :+1: